src/jquery.multiselect.js

```diff
@@ -398,6 +409,9 @@
 
       self._setButtonValue(value, isDefault);
 
+      if ( !/\bbutton\b/.test( options.wrapText ) )
```

Comment:
For consistency I prefer that even single line if statements be wrapped in braces.

Reply:
ok